web/src/pages/NotificationsPage.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <h3 class="text-h3 mb-4">Notifications</h3>
+
+  <AppCard>
+    <v-btn-toggle
+      v-model="showFilter"
+      color="info"
+      group
+      mandatory
+      style="border: 1px #b0b0b0 solid !important; border-radius: 6px"
+      @update:model-value="page = 1"
+    >
+      <v-btn
+        value="All"
+        style="width: 100px; border-left: 1px #b0b0b0 solid"
+        text="All"
+      />
+      <v-btn
+        value="Unread"
+        style="width: 100px; border-left: 1px #b0b0b0 solid"
+        text="Unread"
+      />
+    </v-btn-toggle>
+
+    <v-list
+      class="py-0 mt-4"
+      lines="two"
+    >
+      <v-data-iterator :items="notifications">
+        <template #default="{ items }">
+          <v-list-item
+            v-for="(item, index) of items"
+            :key="index"
+            :value="item"
+            class="py-4 px-8"
+            :base-color="item.raw.readAt ? '' : 'error'"
+            @click="markAsRead(item.raw)"
+          >
+            <template #append>
+              <v-btn
+                v-if="item.raw.href"
+                size="x-small"
+                class="mt-0 ml-2 align-top"
+                icon="mdi-link-variant"
+                :color="!item.raw.readAt ? 'error' : 'success'"
+                @click.stop="openNotification(item.raw)"
+              ></v-btn>
+            </template>
+            <p class="text-subtitle-1 font-weight-retular font-italic textSecondary">
+              {{ formatDate(item.raw.createdAt) }}
+            </p>
+            <div>
+              <h6 class="text-subtitle-1 font-weight-semibold mb-1">{{ item.raw.title }}</h6>
+            </div>
+            <p class="text-subtitle-1 font-weight-regular textSecondary">{{ item.raw.subtitle }}</p>
+          </v-list-item>
+        </template>
+      </v-data-iterator>
+      <RouteQueryPagination
+        v-model="page"
+        v-model:per-page="perPage"
+        :total-count="totalCount"
+      />
```

Comment:
I don't think we want this component anymore, as `v-data-iterator` provides it's own implementation.